src/block_utils.ts

```diff
@@ -7,38 +7,37 @@ import {
     TFile,
 } from "obsidian";
 
+type CacheItem = SectionCache | ListItemCache;
```

Comment:
`CacheItem` is already a type in obsidian.d.ts. Please keep the types as they are.